docs/requirements.txt

```diff
@@ -41,6 +41,7 @@ jinja2==3.1.2
     # via sphinx
 markupsafe==2.1.3
     # via jinja2
+myst-parser==2.0.0
```

Comment:
would this get lost the next time someone does `make pip-compile` due to:
 https://github.com/Unstructured-IO/unstructured/blob/a5ca628/Makefile#L228 ?